playstats/appstat.py

```diff
@@ -111,3 +112,27 @@ def title(self):
         selector = CSSSelector('.id-app-title')
         match = self.tree.xpath(selector.path)
         return match[0].text
+        
+    def keyword_rank(self, search):
```

Comment:
Let's move this to a new file `playstat.py` just like `appstat.py` because this is not related to a particular app but is a general stat.

Reply:
Okay, as a seperate research tool i guess! Optimisation websites offer this simple function for over 50$/month !